src/commands/commit.ts

```diff
@@ -24,11 +28,35 @@ export async function _commit(parsedSettings: ParsedSettings): Promise<void> {
   if (Number.isNaN(newMigrationNumber)) {
     throw new Error("Could not determine next migration number");
   }
-  const newMigrationFilename =
-    String(newMigrationNumber).padStart(6, "0") + ".sql";
-  const currentMigrationPath = getCurrentMigrationPath(parsedSettings);
-  const body = await fsp.readFile(currentMigrationPath, "utf8");
-  const minifiedBody = pgMinify(body);
+
+  // See if we have a message arg
+  const messageIndex =
+    process.argv.findIndex(arg => arg === "--message" || arg === "-m") + 1;
+
+  // If we do, fetch, and replace any whitespace with '_'
+  const messageFromCommandArgs = messageIndex && process.argv[messageIndex];
```

Comment:
We should throw an error if `--message` or `-m` were specified but there was no argument after it.